components/PostCardList/PostTypeContext.ts

```diff
@@ -0,0 +1,8 @@
+import * as React from "react";
+
+export enum PostType {
+  recent,
+  lastUpdate
+}
```

Comment:
What `PostType` means doesn't sound matching what it actually is. "PostType" sounds the type of post. Probably you need to come up with proper name for it.